src/components/regionalization/RegionalizationModal/regionalization-modal.scss

```diff
@@ -0,0 +1,100 @@
+@import "src/styles/scaffold";
+
+[data-modal-overlay] {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 3;
+  background-color: rgb(0 0 0 / 25%);
+}
+
+[data-regionalization-modal] {
+  position: fixed;
+  top: 30vh;
+  right: rem(24px);
+  left: rem(24px);
+  max-width: calc(var(--fs-grid-breakpoint-desktop) / 3);
+  margin: auto;
+  background-color: var(--fs-color-body-bkg);
+  border-radius: var(--fs-spacing-0);
+
+  @include media(">=tablet") {
+    width: calc(100vw / 3);
+    min-width: calc(var(--fs-grid-breakpoint-desktop) / 3);
+  }
+
+  &[data-regionalization-modal-state="in"] {
+    animation: motion-in .2s ease-in;
+  }
+
+  &[data-regionalization-modal-state="out"] {
+    animation: motion-out .2s ease-out;
+  }
+
+  .regionalization-modal__header {
+    padding: var(--fs-spacing-4) var(--fs-spacing-7) var(--fs-spacing-4) var(--fs-spacing-4);
+
+    [data-regionalization-modal-title] {
+      margin-bottom: rem(10px);
+    }
+
+    [data-regionalization-modal-description] {
+      color: var(--fs-color-text-light);
+    }
+
+    .regionalization-modal__button {
+      position: absolute;
+      top: 0;
+      right: 0;
+      background-color: transparent;
```

Comment:
Can we just keep the default background? Otherwise we will lost the focus effect/indicator.

```suggestion
```